Dockerfile

```diff
@@ -44,6 +44,6 @@ ENV HOME /opt/wildfly
 WORKDIR /opt/wildfly/source
 USER    wildfly
 
-EXPOSE 7600 8080 8787 9990 9999
+EXPOSE 8080
```

Comment:
But on the other hand this causes you cannot debug or connect to wildfly console. Wouldn't just moving 8080 to be the first port fix it?